src/node_api.cc

```diff
@@ -156,14 +156,20 @@ class HandleScopeWrapper {
 // across different versions.
 class EscapableHandleScopeWrapper {
  public:
-  explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) : scope(isolate) {}
+  explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) :
+    scope(isolate), _escapeCalled(false) {}
+  bool escapeAlreadyCalled(void) {
+    return _escapeCalled;
+  }
```

Comment:
Style issues: method should be `bool escape_called() const {`, the data member should be `escape_called_`.